src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -172,9 +172,23 @@ private void initCommonVariables(InstructorAttributes instructor, String selecte
             questionToDetailsMap.put(question, questionDetails);
         }
         
+        this.sections = sectionNames();
+        
         displayableFsName = sanitizeForHtml(bundle.feedbackSession.feedbackSessionName);
         displayableCourseId = sanitizeForHtml(bundle.feedbackSession.courseId);
     }
+
+    private List<String> sectionNames() {
```

Comment:
Rename this to getSectionsFromBundle? Looks good to merge after that. Did you check how significant this optimisation is?